src/app/app.module.ts

```diff
@@ -33,6 +37,15 @@ import {
 	SuggestService
 } from './services';
 
+export const firebaseConfig = {
+    apiKey: "AIzaSyBqYyGpkkRu_HLalpEsCcJmka-ZrOsJRog",
```

Comment:
What about adding the config in environment and loading these things from the environment at the time of deployment or local testing?